Gemfile

```diff
@@ -56,7 +56,7 @@ gem 'browse-everything', '~> 0.13.0'
 gem 'mediainfo', git: "https://github.com/avalonmediasystem/mediainfo.git", branch: 'avalon_fixes'
 gem 'rest-client'
 gem 'roo'
-gem 'rubyhorn', git: "https://github.com/avalonmediasystem/rubyhorn.git", tag: 'avalon-r6'
+gem 'rubyhorn', git: "https://github.com/avalonmediasystem/rubyhorn.git", ref: '57c12f23344d8015e223bd36e2544f98fb3a672d'
```

Comment:
This commit allows the matterhorn config file to have ERB in it, see this PR: https://github.com/avalonmediasystem/rubyhorn/pull/11